openff/toolkit/tests/test_forcefield.py

```diff
@@ -4907,6 +4907,32 @@ def test_read_smirnoff_spec_freesolv(
         )
 
 
+class TestForceFieldGetPartialCharges:
+    def get_partial_charges(self, mol, ff):
```

Comment:
Took me a sec to understand the use of this method. Since it's not in the public API we can make the name really big. 

```suggestion
    def get_partial_charges_from_create_openmm_system(self, mol, ff):
```